mne/tests/test_event.py

```diff
@@ -22,6 +23,10 @@
 fname_txt = op.join(base_dir, 'test-eve.eve')
 fname_txt_1 = op.join(base_dir, 'test-eve-1.eve')
 
+# for testing Elekta averager
+fname_raw_elekta = op.join(base_dir, 'trigtest_3ch_raw.fif')
+fname_ave_elekta = op.join(base_dir, 'test_elekta_ave.fif')
```

Comment:
the test files are missing.

would it just be possible to fake a acq_pars for the sample data?